### ISSUE TITLE
Monitor refresh rate changes when I close the game

### ISSUE THREAD

**Issue by Hexronimo** [libgdx/libgdx#6010]
Hello! 
I have made a desktop project with gdx-setup.jar with Box2D extension. When I run an application everything seems to be ok:  window with Badlogic logo opens as expected; but when I close it, the monitor's screen turns black for a part of a second and then shows again but pixelated. 
It seems like libgdx application when I close it changes my monitor refresh rate from 59,95 Hz to 59,88 Hz (probably it's the first/lowest rate from available).  And everything happens again when I run and close application.

- It happens both when I run the game from IDE or from the runnable jar. 
- I use Ubuntu 18.04.
- It happens with predefined "game" which has only windowed mode I didn't add any window settings. 
- In the syslog I see:
"Apr 21 15:06:04 NVIDIA(0): Setting mode "DVI-D-1: 1920x1200_60_0 @1920x1200 +0+0 {ViewPortIn=1920x1200, ViewPortOut=1920x1200+0+0}"
Apr 21 15:06:04 NVIDIA(GPU-0): Display (DELL U2410 (DFP-2)) does not support NVIDIA 3D Vision
Apr 21 15:06:04 NVIDIA(GPU-0):     stereo.
... 
Apr 21 15:06:16 NVIDIA(0): Setting mode "DVI-D-1: nvidia-auto-select @1920x1200 +0+0 {ViewPortIn=1920x1200, ViewPortOut=1920x1200+0+0}""

Maybe the problem is with my X settings because I can't find the other mentions of this problem on the Internet, but can I put something to game configuration to prevent it from changing my monitor refresh rate (I just need to implement simple 2d windowed desktop game)? It's just my first day of use of libgdx, but I can't concentrate on learning it when I need to fix monitor settings after each application run.

**Comment by NathanSweet** [comment]
Could maybe try other graphics drivers?

**Comment by mgsx-dev** [comment]
it sounds like a X/nvidia config issue or bug, might be similar to [this thread](https://askubuntu.com/questions/776557/how-to-prevent-nvidia-auto-select-switch-to-incorrect-resolution-at-login-screen)

**Comment by Hexronimo** [comment]
Yes, it seems like it pure X config problem, when I run xranrd it shows the monitor's supported resolutions:
1920x1200     59.95*+  59.88  
1600x1200     60.00  
1280x1024     75.02    60.02  
etc.
The refresh rate reset has appeared only in 1920x1200 all other works fine. Maybe I need to get rid of this 59.88 which mode somehow. But I'm totally not in the mood to fight with X now, usually, it causes more problems.  Sorry for wasting your time and thanks for the help. At least now I know the problem is with my PC.
